spec/support/vcr_helper.rb

```diff
@@ -0,0 +1,7 @@
+def fix_token_expires_at(interaction)
```

Comment:
You might need to wrap this in a namespace of Spec::Support:VcrHelper for zeitwerk.

Reply:
Okay, we'll need to fix the other files in `spec/support/` as well then